clients/src/main/java/org/apache/kafka/common/security/authenticator/SaslServerAuthenticator.java

```diff
@@ -395,6 +454,46 @@ private void handleSaslToken(byte[] clientToken) throws IOException {
         }
     }
 
+    private long calcSessionLifetimeAndMaybeAuthenticationEndAndSessionExpirationTimes() {
```

Comment:
nit: Can we reduce the length of the method name?

Reply:
👍 fixed